### ISSUE TITLE
Problem to use custom model for Farsi lang

### ISSUE THREAD

**Issue by AliRezaSafaei9494** [JaidedAI/EasyOCR#752]
Hi,
thanks for such great repo

i have followed and did all the instructions exactly the same as [custom_model.md](https://github.com/JaidedAI/EasyOCR/blob/master/custom_model.md)

for log_train.txt:
[180000/300000] Train loss: 0.03544, Valid loss: 0.17261, Elapsed_time: 43124.35771
Current_accuracy : 88.976, Current_norm_ED  : 0.9628
Best_accuracy    : 89.210, Best_norm_ED     : 0.9632
--------------------------------------------------------------------------------
Ground Truth              | Prediction                | Confidence Score & T/F
--------------------------------------------------------------------------------
میشه                      | میشه                      | 0.9913	True
شماره                     | شماره                     | 0.9978	True
--------------------------------------------------------------------------------

and
easyocr.Reader(['fa'], recog_network='custom_example')

but i  got nonsense result 
even for training data

**Comment by manjunath7472** [comment]
How did u get log train.txt file? Where it ll store

**Comment by uniquefan** [comment]
>/EasyOCR/trainer/saved_models/experiment_name
